scraper/src/mindtouch2zim/entrypoint.py

```diff
@@ -47,15 +36,13 @@ def add_zim_config_flags(parser: argparse.ArgumentParser, defaults: "ZimConfig")
 
     parser.add_argument(
         "--publisher",
-        help=f"Publisher name. Default: {defaults.publisher!r}",
-        default=defaults.publisher,
+        help=f"Publisher name. Default: {Context.publisher!r}",
     )
 
     parser.add_argument(
         "--file-name",
         help="Custom file name format for individual ZIMs. "
-        f"Default: {defaults.file_name!r}",
-        default=defaults.file_name,
+        f"Default: {Context.file_name!r}",
```

Comment:
Don't use `!r` which calls `__repr__` but `!s` which calls `__str__` instead. repr is meant for _internal_ representation of the object. str is the human-readable one.